db/seeds.rb

```diff
@@ -19,8 +10,20 @@
   password_confirmation: 'password'
```

Comment:
so by default anyone can log in as ben? seems kind insecure for a public repo and public site.